react-components/oodt_opsui_sample_app/src/components/SimpleSnackBar.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import Snackbar from "@material-ui/core/Snackbar";
+import SnackbarContent from "@material-ui/core/SnackbarContent";
+import Typography from "@material-ui/core/Typography";
+import InfoIcon from "@material-ui/icons/Info"
+import WarningIcon from "@material-ui/icons/Warning"
+import SuccessIcon from "@material-ui/icons/CheckCircleOutline"
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from "@material-ui/icons/Close";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core";
+
+const styles = theme => ({
+  close: {
+    padding: 10
+  },
+  info: {
+    backgroundColor: "#0F8EF7",
+  },
+  warning: {
+    backgroundColor: "#F0AD11"
+  },
+  success: {
+    backgroundColor: "#41BE41"
+  },
+  error: {
+    backgroundColor: "red"
+  },
+  snackContent: {
+    display: "flex",
+    height: "2vh",
+    alignItems: "center",
+  },
+  icon: {
+    paddingRight: "2%",
+    fontSize: "1.5em"
+  }
+});
+
+class SimpleSnackBar extends React.Component {
```

Comment:
We could use a package like https://www.npmjs.com/package/notistack to get snackbar support as well. It provides a hook `useSnackbar()` which is convenient. WDYT?

Reply:
Using hooks wouldn't be possible as all components in our app are class components. However, that library can be used in class components in another way. I will look into it.

Reply:
Implemented snack bars in notistack and removed the `SimpleSnackBar` component.